src/Config.php

```diff
@@ -33,6 +33,7 @@ public function getDefaultConfig()
             'run_scripts' => 1,
             'security_updates_only' => 0,
             'number_of_concurrent_updates' => 0,
+            'allow_security_updates_on_concurrent_limit' => 1,
```

Comment:
This should now default to 0 to not change current behavior 🤓

Reply:
I have switched per request, pushed changed: https://github.com/violinist-dev/violinist-config/compare/dd534c8b96d6aa30017b5e5b02ba66397f5843af..18a011b2cbd06d9338ee93b30f9582db664a7b26